src/store/reducers/singleCohort.reducer.js

```diff
@@ -0,0 +1,32 @@
+import { SINGLE_COHORT } from '../actions/actions.type';
+
+export const INITIAL_STATE = {
+  isSingleCohortPending: false,
+  isSingleCohortFailure: false
```

Comment:
I believe you can call those "isPending" and "isFailure" as when we will call this data from the store it will be `singleCohort.isLoading` and `singleCohort.isFailure`